src/tools/id-mapping/components/IDMappingForm.tsx

```diff
@@ -354,7 +354,9 @@ const IDMappingForm = () => {
                     });
                   }}
                   label={fromDbInfo.displayName}
-                  defaultActiveNodes={[fromDb.selected]}
+                  defaultActiveNodes={
+                    fromDb.selected ? [fromDb.selected.toString()] : undefined
```

Comment:
Why the `toString()` here?

Reply:
Good question, it's not needed. I have used `as string` instead.